ia-configs/IA.GLOBALS.json

```diff
@@ -1,17 +1,13 @@
 {
-    "_global_endpoints": {
-        "_note": "The common WBIA endpoints targeted by all WBIA configs.",
-        "add_images": "/api/image/json/",
-        "add_annotations": "/api/annot/json/",
-        "start_identify": "/api/engine/query/graph/",
-        "identify_review": "/api/review/query/graph/",
-        "start_detect": {
```

Comment:
I have updated the Sage image to combine this into a single API, `/api/engine/detect/cnn/`.  Houston can specify `algo` as a parameter to use either `lightnet` or `yolo`.